docs/howto/kubernetes/gardener/create-shoot-cluster.md

```diff
@@ -0,0 +1,111 @@
+# Create Kubernetes cluster using Cleura Cloud Control Panel 
```

Comment:
Let's use "Creating Kubernetes clusters" here (following the structure we are using elsewhere).